lib/github-api/createDiscussionRepo.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * createDiscussionRepo github-api method used to create a discussion repo for a user/org in probot-background-check org
+ * @param {object} github - authenticated github client
+ * @param {object} data
+ * @param {string} data.owner - name of user/org for whose repo's will be scanned for toxic users and their discussion will be held in owner-discussions repo.
+ * @returns {Promise<object>}
+ */
+
+module.exports = (github, { owner }) => {
+  return github.repos.createForOrg({
+    org: 'probot-background-check',
+    name: `${owner}-discussions`,
+    description: 'Repo to have discussions about toxic users',
```

Comment:
Lets change this to something a little less attacking; calling users toxic is setting the standard for how they'll be talked about. How about:

> Repo to discuss users who have contributed to {{ repo }} and have been toxic.

Its a subtle change, but saying they've "been toxic" is better than calling them toxic. Its a description of their actions, not of their person.

Reply:
nice suggestion. I guess I should do "been hostile" as per @gr2m origin idea description